CHANGELOG.released.md

```diff
@@ -7,6 +7,48 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Calendar Versioning](http://calver.org/) `0Y.0M.MICRO`.
 For upgrade instructions, please check the [migration guide](MIGRATIONS.released.md).
 
+## [20.05.0](https://github.com/scalableminds/webknossos/releases/tag/20.05.0) - 2020-05-05
+[Commits](https://github.com/scalableminds/webknossos/compare/20.04.0...20.05.0)
```

Comment:
Maybe add a highlights section? Those are usually the changes that I mentioned during release mails as well.